pkg/transformer/openshift/openshift.go

```diff
@@ -226,6 +373,10 @@ func (o *OpenShift) Transform(komposeObject kobject.KomposeObject, opt kobject.C
 
 		allobjects = append(allobjects, objects...)
 	}
+
+	if hasBuild {
+		logrus.Infof("Buildconfig using %s::%s as source.", buildRepo, opt.BuildBranch)
```

Comment:
This is intended! This log needs to be printed only once during runtime, if there's even a single build in the compose file.